app/_locales/en_GB/messages.json

```diff
@@ -434,7 +434,7 @@
     "message": "The token's on-chain history reveals prior instances of suspicious airdrop activities."
```

Comment:
No need to check this file matches `app/_locales/en/messages.json` as the `yarn verify-locales:fix` automates copying the changes.